slurm/src/slurmcc/util.py

```diff
@@ -196,19 +196,11 @@ def is_autoscale_enabled(subprocess_module: Optional[Any] = None) -> bool:
         subprocess_module = subprocess_module_tmp
```

Comment:
Let's remove the subprocess_module argument and and reference in this method to subprocess. I did a git grep, and we aren't actually ever calling this function with a mock subprocess module or anything.

Reply:
Done